extensions/arc/README.md

```diff
@@ -0,0 +1,17 @@
+# Microsoft Azure Arc Extension for Azure Data Studio
+
+Welcome to Microsoft Azure Arc Extension for Azure Data Studio!
```

Comment:
Are you planning on getting PMs to help with the naming and descriptions for this extension? It's a little plain right now

Reply:
Yes, I'll make it better with their input